samples/meta-body-tracking-sample/README.md

```diff
@@ -0,0 +1,10 @@
+# Meta Body Tracking Sample
+
+> Note: this project requires Godot 4.3 or later
+
+This is a sample project demonstrating tracking features on Meta headsets. This includes body / hand tracking, which is supported on Quest 2, Quest 3, and Quest Pro;
+as well as face tracking, which is only supported on Quest Pro. For more Meta-specific hand tracking features, check out the [Meta Hand Tracking Sample](https://github.com/devloglogan/godot_openxr_vendors/tree/sample-readme-files/samples/meta-hand-tracking-sample).
```

Comment:
The link to the meta hand tracking sample is pointing to your branch instead of the documentation page.

Reply:
Whoops, fixed the link to point to the hand tracking sample project on upstream master.